src/Lecture1.hs

```diff
@@ -123,4 +136,13 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = let (l, g) = go n list in fillTemplate n l g
```

Comment:
I see GHC warnings about defaulting constraints ⚠️ 

This happens here because numeric literals are polymorphic in Haskell and due to type inference the return values of `go` are inferred polymorphic as well. You can fix this problem by specifying additional type signatures.

Reply:
Thanks for taking the time to review these. 
Considering it's a warning, is there an issue or something that can go wrong as a result of this?

Reply:
In this particular case, probably nothing 🙂 
Generally, you might get unexpected behaviour or decreased performance due to this warning so it's better to avoid ambiguous typing